tests/integration/mixer/util.go

```diff
@@ -51,7 +51,7 @@ func GetReporterCodeLabel() string {
 func VisitProductPage(ingress ingress.Instance, timeout time.Duration, wantStatus int, t *testing.T) error {
 	start := time.Now()
 	for {
-		response, err := ingress.Call("/productpage")
+		response, err := ingress.Call("/productpage", "" /* host */)
```

Comment:
It looks like the parameter order needs to be reversed. Using the Echo-like API here makes things more explicit.

Reply:
Done.